apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -1057,6 +1072,14 @@ export class MarkdownDocumenter {
       }
     }
 
+    if (ApiAbstractMixin.isBaseClassOf(apiItem)) {
+      if (apiItem.isAbstract) {
+        section.appendNode(
+          new DocParagraph({ configuration }, [new DocCodeSpan({ configuration, code: 'abstract' })])
+        );
+      }
+    }
```

Comment:
I'm not sure whether it was intentional or not, but the order of listed modifiers was previously in alphabetical order. This breaks that - maybe we can show `abstract` first above the other modifiers to keep with alphabetical ordering?

Reply:
I didn't realize they were in alphabetical order, and I'm too not sure whether this was coincidence or intentional. If we keep the additional table at all (see above), of course we can keep the alphabetical order. But what would really be nice here would be a way to _configure_ the desired order in api-documenter.

Reply:
I thought about that and have the following suggestion: Wouldn't it make sense to use the modifier order that would be correct in TypeScript code? Unlike other languages like Java, in TypeScript, the order of modifiers is defined strictly, and unfortunately, it is not alphabetical order. In that sense, the current order is already wrong. It seems the correct order is

> (`public` | `protected` | `private`)? (`static`? | `abstract`?) `override`? `readonly`?

(No member can be `static` and `abstract` at the same time, so the order of these two does not matter.)
_api-documenter_ currently only uses `protected`, `static`, `abstract` (new in this PR) and `readonly`, so `public` (default!), `private` (no API!) and `override` (maybe interesting for navigation, but not as a modifier) do not apply.

I'll add a commit changing that and then we can see whether we keep it that way.

Reply:
Btw., in my first comment, I misunderstood you were referring to the order of _member tables_, not to the order of modifiers. So now that I realized what you meant, we still need to discuss whether to add an "Abstact Classes" table at all and if so, where / in what order.

Reply:
> I thought about that and have the following suggestion: Wouldn't it make sense to use the modifier order that would be correct in TypeScript code?

Interesting idea - I think either this or alphabetical both make sense!